openedx/core/djangoapps/programs/tasks.py

```diff
@@ -105,6 +102,35 @@ def get_certified_programs(student: User, raise_on_error: bool = False) -> List[
     return certified_programs
 
 
+def get_revokable_program_uuids(course_specific_programs: List[Dict], student: User) -> List[str]:
```

Comment:
No changes here, just relocated. I'm trying to group the utility functions together with other utility functions. This leaves all the celery tasks grouped together at the bottom of the file. This makes it more readable _to me_, but please yell if you think this makes it worse :)